Dockerfile

```diff
@@ -52,6 +52,11 @@ ARG NODEJS_VERSION=22.19.0
 
 ARG PNPM_VERSION=10.15.0
 
+ARG YARN_VERSION=1.22.22
```

Comment:
Please add the Renovate comments so that Renovate can keep all these 3 dependencies updated.